magstock/static/food_menu.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>MAGStock Menu</title>
+    </head>
+    <body>
+        <h2 align="center">MAGStock Food Service</h2>
+        People who purchase food will be provided 3 meals per day, served in the pavillion:
+        <ul>
+            <li>
+                <b>10am Friday (breakfast)</b> <br/>
+                Eggs, Sausage, and Hash Browns
+            </li>
+            <li>
+                <b>2pm Friday (lunch)</b> <br/>
+                TBA
+            </li>
+            <li>
+                <b>7pm Friday (dinner)</b> <br/>
+                Mexican
+            </li>
+            <li>
+                <b>10am Saturday (breakfast)</b> <br/>
+                Pancakes
+            </li>
+            <li>
+                <b>2pm Saturday (lunch)</b> <br/>
+                TBA
+            </li>
+            <li>
+                <b>7pm Saturday (dinner)</b> <br/>
+                Kung Pao Chicken
+            </li>
+            <li>
+                <b>10am Sunday (breakfast)</b> <br/>
+                Limited options such as granola bars and such.
+            </li>
+        </ul>
+    </body>
+</html>
```

Comment:
Below you can see where we checked in MAGStock images to override the default themed images.